sled-agent/src/metrics.rs

```diff
@@ -161,6 +168,16 @@ async fn metrics_task(
                 add_datalink(&log, &mut tracked_links, &kstat_sampler, link)
                     .await;
             }
+            Message::TrackCpu => {
```

Comment:
I wasn't exactly sure how to handle this. At the moment, we create one target, which checks cpu stats for all cores on the sled, and the core number is a metric field. Alternatively, we could make a target per core, and move the core number to the target. This implementation is a bit simpler, and it matches the approach we take to instrument cpu usage in propolis. But we could switch to a target per core if that makes more sense semantically within oximeter.